src/SnehalW/ArrayEquality.java

```diff
@@ -0,0 +1,34 @@
+package SnehalW;
+
+public class ArrayEquality 
+{
+	int array1[] = {21, 57, 11, 37, 24};
+	int array2[] = {21, 57, 11, 37, 24};
+	int count;
+	void equality()
+	{
+		for(int i=0;i<array1.length;i++)
+		{
+			for(int j=0;j<array2.length;j++)
+			{
```

Comment:
Hi @SnehalWAug19 
Here good you to see you are comparing the length of both array..
best way to compare both's length before the for loop.. i mean 
if(both's lnegth is same){
     //Add your code
}else{
   SOP("Length is miss match");
}

this is correct just anothere way..

Reply:
thanks harsh